packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -482,7 +482,7 @@ export class PeerManager {
   private async requestMetadata(peer: PeerId): Promise<void> {
     const peerIdStr = peer.toString();
     try {
-      this.logger.warn("requestMetadata", {peer: prettyPrintPeerIdStr(peerIdStr)});
+      this.logger.debug("requestMetadata", {peer: prettyPrintPeerIdStr(peerIdStr)});
```

Comment:
I think as per suggestion from @wemeetagain we might wanna completely remove this, I tend to agree